lib/util/buttons/custom_banner.dart

```diff
@@ -8,7 +8,8 @@ class CustomBanner extends StatelessWidget {
 
   const CustomBanner(this.text, {super.key, this.displayColorKind = DisplayColorKind.primary});
 
-  factory CustomBanner.error(text) => CustomBanner(text, displayColorKind: DisplayColorKind.error);
+  factory CustomBanner.error(text, {Key? key}) =>
+      CustomBanner(text, displayColorKind: DisplayColorKind.error, key: key);
   factory CustomBanner.warning(text) => CustomBanner(text, displayColorKind: DisplayColorKind.warning);
```

Comment:
why no key here?

Reply:
Not needed yet... If you want I can add it for consistency though :)